File: Reseau/Evaluation2/Serveur_Villages/PacketCom.cpp
#include "PacketCom.h"

PacketCom::PacketCom(){
    this->type = NULL;
    this->object = NULL;
}

void PacketCom::setType(const char* type)
{
    this->type = new char[strlen(type) + 1];
    strcpy(this->type, type);
}

const char* PacketCom::getType()
{
    return this->type;
}

void PacketCom::setObject(void* object)
{

}

void* PacketCom::getObject()
{

}

File: Reseau/Evaluation3/RMP/src/Protocole/PacketCom.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package Protocole;

import java.io.Serializable;


public class PacketCom implements Serializable{
    private String type;
    private Object objet;

    public PacketCom(String type, Object objet){
        this.type = type;
        this.objet = objet;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public Object getObjet() {
        return objet;
    }

    public void setObjet(Object objet) {
        this.objet = objet;
    }
}
